core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1530,7 +1577,7 @@ private void printAssignments(NodeRepresentation nodeOutput, Assignments assignm
                     // skip identity assignments
                     continue;
                 }
-                nodeOutput.appendDetails("%s := %s", entry.getKey(), unresolveFunctions(entry.getValue()));
+                nodeOutput.appendDetails("%s := %s", anonymizer.anonymize(entry.getKey()), anonymizer.anonymize(unresolveFunctions(entry.getValue())));
             }
         }
 
```

Comment:
Please make sure we handle `visitTableFunction` as well, if needed, from latest changes